def add_accuracy_data(self):
    h2o = mysql.connector.connect(client_flags=[ClientFlag.LOCAL_FILES], user='root', password='0xdata', host='172.16.2.178', database='h2o')
    cursor = h2o.cursor()
    try:
        cursor.execute("LOAD DATA LOCAL INFILE '../h2o-test-accuracy/src/test/resources/accuracyDataSets.csv' INTO TABLE AccuracyDatasets COLUMNS TERMINATED BY ',' LINES TERMINATED BY '\n' IGNORE 1 LINES;")
        h2o.commit()
    except:
        traceback.print_exc()
        h2o.rollback()
        assert False, 'Failed to add accuracy test cases to h2o database!'
    cursor.close()
    h2o.close()
